Update README & Move app.py to root (#7)
Lightning-AI/LAI-TLDR: README.md

<div align="center">
    <h1>
        <img src="https://lightningaidev.wpengine.com/wp-content/uploads/2022/11/image-6.png">
        <br>
        TL;DR with Lightning
        </br>
    </h1>

<div align="center">

<p align="center">
  <a href="#run">Run</a> •
  <a href="https://www.lightning.ai/">Lightning AI</a> •
  <a href="https://lightning.ai/lightning-docs/">Docs</a>
</p>

[![ReadTheDocs](https://readthedocs.org/projects/pytorch-lightning/badge/?version=stable)](https://lightning.ai/lightning-docs/)
[![Slack](https://img.shields.io/badge/slack-chat-green.svg?logo=slack)](https://www.pytorchlightning.ai/community)
[![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Lightning-AI/lightning/blob/master/LICENSE)

</div>
</div>

______________________________________________________________________

> Use TLDR component to pre-train or fine-tune a large language model for text-summarization, 

Reviewer: ```suggestion
Use TLDR to pre-train or fine-tune a large language model for text summarization, 
```

> with as many parameters as you want (up to billions), using multiple GPUs, across multiple machines, 
> on your own data, and all without any infrastructure hassle! All handled easily with the [Lightning Apps framework](https://lightning.ai/lightning-docs/).

Reviewer: ```suggestion
with as many parameters as you want (up to billions!). 

You can do this:
*using multiple GPUs
*across multiple machines
*on your own data
*all without any infrastructure hassle! 

All handled easily with the [Lightning Apps framework](https://lightning.ai/lightning-docs/).
```

## Run

> To run this app, paste the following code snippet in a file `app.py`:

Reviewer: ```suggestion
To run TLDR, paste the following code snippet in a file `app.py`:
```
